### PR TITLE
Benchmark changes

### PR DESCRIPTION
**Pull request by FilipAndersson245**
# Description

This PR does a total of 3 things,
1. It fixes an error when running the `cargo bench` suit where nushell constants where not set correctly ending in an error when running the code.
2. It removes 2 redundant benchmark runs as these where duplicates of existing ones.
3. It reduced encoding and decoding benchmark suit future, only having 4 benches instead of the previous 8.

# User-Facing Changes

# Tests + Formatting

# After Submitting

### CODE REVIEW

File: benches/benchmarks.rs [hunk 1]
```diff
@@ -143,44 +121,46 @@ fn encoding_test_data(row_cnt: usize, col_cnt: usize) -> Value {
...
-                let mut res = vec![];
-                let test_data = PluginOutput::CallResponse(
-                    0,
-                    PluginCallResponse::value(encoding_test_data(row_cnt, col_cnt)),
-                );
-                let encoder = EncodingType::try_from_bytes(fmt.as_bytes()).unwrap();
-                b.iter(|| encoder.encode(&test_data, &mut res))
-            });
-        }
+    let row_cnt = 10000;
```
Comment (fdncred) [inline benches/benchmarks.rs:124]: why are these row_cnt and col_cnt hard coded?

Comment (fdncred) [reply benches/benchmarks.rs:124]: oh wait, they were hard coded to begin with but there were several benchmarks, now there's just 1 with 10k and 15. Maybe we should at least keep 2, one small and one bigger?

Comment (FilipAndersson245) [reply benches/benchmarks.rs:124]: Previously we stored these in a vec and iterated over it, now I removed the other so its only 1 left. And I thought it fine to have it hard coded as its just used for the benchmark.

Comment (FilipAndersson245) [reply benches/benchmarks.rs]: Fixed, but i still think its quite a lot at least as most of the current bench suit is encoding and decoding messagepack and json.

**Comment by fdncred** [comment]
Thanks

**Pull request merged**